testing/inputs/categories.csv

```diff
@@ -0,0 +1,391 @@
+"ID","Category"
```

Comment:
Please remove or adapt to architecture.
As a start this file can be probably empty.